css/styles.css

```diff
@@ -0,0 +1,93 @@
+body {
+  background-color: #e5e5e5;
+  margin: 0;
+  padding: 0;
+}
+
+.wrapper {
+  display: flex;
+  width: 375px;
+  height: 718px;
+  margin: auto;
+  background-color: white;
+  border-bottom-left-radius: 50px;
+  padding: 0;
+}
+
+.nav-bar {
+  margin: 26px 24px 20px;
+  color: #6070ff;
+  display: flex;
+  font-weight: 700;
+  justify-content: space-between;
+}
+
+.nav-bar p {
+  padding: 0;
+  margin: 0;
+  font-size: 18px;
+  font-weight: 700;
+}
+
+section {
+  width: 375px;
+}
+
+.nav-bar li {
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+}
+
+.second-section-container {
+  background-image: url("/assets/images/header-shapes\ mobile.png");
+  background-size: 100% 100%;
+}
```

Comment:
- [ ] You made a mistake while you were trying to upload the background image, please fix it.